models/sessions.go

```diff
@@ -92,7 +92,7 @@ func FindSessionById(tx *storage.Connection, id uuid.UUID) (*Session, error) {
 
 func FindSessionByUserID(tx *storage.Connection, userId uuid.UUID) (*Session, error) {
 	session := &Session{}
-	if err := tx.Eager().Q().Where("user_id = ?", userId).First(session); err != nil {
+	if err := tx.Eager().Q().Where("user_id = ?", userId).Order("created_at asc").First(session); err != nil {
```

Comment:
Whenever adding ordering, can you make sure there's an index on the column. There can be potentially thousands of sessions (e.g. ones that have not been cleared even though no longer used).

Reply:
AFAIK this functions is only used in tests though  but will add it in together with the (factor_id, user_id) index that we're adding

Reply:
Will do!

Reply:
TIL: I don't think you can use  test functions in one package when writing tests for another package

https://github.com/golang/go/issues/39565

Reply:
I will add the index -- there's a good chance we'll need to method in the future on the user side as well